cache/lru_test.go

```diff
@@ -0,0 +1,60 @@
+package cache_test
+
+import (
+	"testing"
+
+	lru "github.com/TheAlgorithms/Go/cache"
```

Comment:
This is your change. There is no big dependency on the name `lru` for you to add `lru` here. Using shortcuts is not a good practice - fixing is simply a version of `gofmt -r 'lru.a-> cache.a' -s -w path/to/this/file.go`

Reply:
I fixed by rename function `New` to `NewLRU` for using by `cache.NewLRU()`